### PR TITLE
 Increased the Prow timeout for the e2e tests in the CLO and OAL repos from 2h to 3h

### PR DESCRIPTION
**Pull request by syedriko**

### CODE REVIEW

File: ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml [hunk 1]
```diff
@@ -86,6 +86,7 @@ presubmits:
     decorate: true
     decoration_config:
       skip_cloning: true
+      timeout: 4h0m0s
```
Comment (jcantrill) [inline ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml:89]: Can we try a smaller jump to say 3 hours?

**Comment by jcantrill** [comment]
Need to resubmit after https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20973/pull-ci-openshift-release-master-generated-config/1423767631357284352#1:build-log.txt%3A9

**Comment by syedriko** [comment]
> Need to resubmit after https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20973/pull-ci-openshift-release-master-generated-config/1423767631357284352#1:build-log.txt%3A9

`make jobs` removes my change with the timeout. What's the schema it's working off?

File: core-services/prow/02_config/_config.yaml [hunk 2]
```diff
@@ -196,6 +196,12 @@ plank:
   - config:
       timeout: 6h0m0s
     repo: openshift/osde2e
+  - config:
+      timeout: 3h0m0s
```
Comment (alvaroaleman) [inline core-services/prow/02_config/_config.yaml:200]: The default timeout for this is 4h, you are reducing it

Comment (syedriko) [reply core-services/prow/02_config/_config.yaml:200]: In that case we need to tweak a different timeout. @alvaroaleman, can you guide us here?
Here's an example of  what we need to fix: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-logging-operator/1169/pull-ci-openshift-cluster-logging-operator-release-5.2-e2e-gcp/1437765835715252224 died saying
```
 {"component":"entrypoint","file":"prow/entrypoint/run.go:165","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 2h0m0s timeout","severity":"error","time":"2021-09-14T15:42:55Z"} 
```

Comment (alvaroaleman) [reply core-services/prow/02_config/_config.yaml:200]: https://docs.ci.openshift.org/docs/architecture/timeouts/#step-registry-test-process-timeouts

Comment (syedriko) [reply core-services/prow/02_config/_config.yaml:200]: I was going by this document https://docs.ci.openshift.org/docs/architecture/timeouts/#prow-test-process-timeouts

**Comment by syedriko** [comment]
The new timeouts appear to be working. The CLO e2e tests ran for > 2h without getting terminated: about 2h3m as the ci/rehearse/openshift/cluster-logging-operator/master/e2e-gcp job
```
[36mINFO[0m[2021-09-15T04:58:02Z] Running step e2e-gcp-test.                   
[36mINFO[0m[2021-09-15T07:00:52Z] Logs for container test in pod e2e-gcp-test: 
```
and about 2h23m as the ci/rehearse/openshift/origin-aggregated-logging/master/cluster-logging-operator-e2e job:
```
[36mINFO[0m[2021-09-15T05:01:51Z] Running step cluster-logging-operator-e2e-test. 
[36mINFO[0m[2021-09-15T07:24:31Z] Logs for container test in pod cluster-logging-operator-e2e-test: 
```
I don't think failures of the e2e tests should keep us from merging this PR.

**Review by jcantrill — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**